airgun/views/cloud_insights.py

```diff
@@ -0,0 +1,89 @@
+from wait_for import wait_for
+from widgetastic.widget import Checkbox
+from widgetastic.widget import Text
+from widgetastic.widget import TextInput
+from widgetastic.widget import View
+from widgetastic_patternfly4 import Button
+from widgetastic_patternfly4 import Pagination
+from widgetastic_patternfly4.button import Button as Pf4Button
+from widgetastic_patternfly4.modal import Modal
+from widgetastic_patternfly4.table import PatternflyTable
+
+from airgun.views.common import BaseLoggedInView
+from airgun.views.common import SearchableViewMixin
+from airgun.widgets import InventoryBootstrapSwitch
+
+
+class RemediationView(Modal):
+    """ Remediation window view"""
+
+    remediate = Pf4Button(locator='//button[text()="Remediate" and @type="submit"]')
+    cancel = Pf4Button(locator='//button[text()="Cancel"]')
+    table = PatternflyTable(
+        locator='.//table[@aria-label="remediations Table"]',
+        column_widgets={
+            'Hostname': Text('./a'),
+            'Recommendation': Text('./a'),
```

Comment:
both locators are same is this working for you?

Reply:
column_widgets should apply that locator only in the context of the column with the name matching the key from the dict, so it shouldn't cross locator text fields in other columns.